client/protoc-gen-improbable-grpc-web/bin/protoc-gen-improbable-grpc-web

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+require('../lib/index');
```

Comment:
What is the purpose of this file?

Reply:
It's how node/npm manages binaries. This file will be automagically `chmod +x`'d when you npm install allowing the end user to invoke it like a binary.  You can also invoke these bins via npm scripts (defined inside the package.json) or via the [`npx` command](https://www.npmjs.com/package/npx).